docs/erwin-goals.md

```diff
@@ -0,0 +1,20 @@
+II a dBildung-IAM as a production ready, stand-alone system
+
+Erwin Team Goals: <br />dBildung-IAM can be used as IDP for Service Providers like LMS and school portals i.e. dBildungscloud and "Schulportal SH":<br />
+1. separating IDM from dBildungscloud supporting the strategic goal to realize a stand-alone system which is production ready.
+2. extend the IDM user's data model with structural information supporting the strategic goal to base the IDM on a German educational data model.
+3. dBildung-IAM can import/sync from other IDPs.
+4. dBildung-IAM can be used as the leading system.
+5. Configuration concerning IDPs, sync, etc. that is currently done via SHD or directly in DB is done via a micro service hosted together with dBildung-IAM  (instead of in dBildungscloud).
```

Comment:
```suggestion
5. Configuration concerning IDPs, sync, etc. that is currently done via Super Hero Dashboard (SHD) or directly in the database is done via a micro service hosted together with dBildung-IAM  (instead of in dBildungscloud).
```